### PR TITLE
Focus on claim when getting semantics, submit better queries to KGTK

### PR DESCRIPTION
**Pull request by elizlee**

### CODE REVIEW

File: cdse_covid/semantic_extraction/run_wikidata_linking.py [hunk 1]
```diff
@@ -44,11 +44,20 @@ def get_best_qnode_for_mention_text(
...
+
+    # Locate the AMR node label associated with the mention text
     for node in amr.nodes:
         token_list_for_node = amr.get_tokens_from_node(node, alignments)
-        if claim_variable_last_token in token_list_for_node:
-            variable_node_label = amr.nodes[node]
+        # Try to match the last token since it is most likely to point to the correct node
+        if claim_variable_tokens[0] in token_list_for_node:
+            variable_node_label = amr.nodes[node].strip('"')
+            # print(f"Variable node label: {variable_node_label}")
```
Comment (joecummings) [inline cdse_covid/semantic_extraction/run_wikidata_linking.py:60]: Can you remove this print statement, too?

File: cdse_covid/semantic_extraction/utils/amr_extraction_utils.py [hunk 2]
```diff
@@ -117,15 +119,30 @@ def create_node_to_token_dict(amr: AMR, alignments: List[AMR_Alignment]) -> Dict
     return {node: " ".join(token_list) for node, token_list in nodes_to_token_lists.items()}
 
 
+def remove_trailing_stop_words(text: str) -> Optional[str]:
```
Comment (joecummings) [inline cdse_covid/semantic_extraction/utils/amr_extraction_utils.py:122]: Are there no instances of preceding stop words?

**Review by joecummings — CHANGES_REQUESTED** [review]
Couple of comments

File: cdse_covid/semantic_extraction/run_wikidata_linking.py (hunk 1 above)
Comment (elizlee) [reply cdse_covid/semantic_extraction/run_wikidata_linking.py:60]: Oops, yeah thanks

File: cdse_covid/semantic_extraction/utils/amr_extraction_utils.py (hunk 2 above)
Comment (elizlee) [reply cdse_covid/semantic_extraction/utils/amr_extraction_utils.py:122]: I should probably rename that since it removes preceding ones too.

**Comment by elizlee** [comment]
> Are there any stats you can provide on _how_ much better these queries work?

This biggest thing that this fixes is the "looking at the entire sentence for an event" issue. This leads to detecting less events, but it is no longer giving events when they're not in the claim itself.

When looking at 20 claim results from the original system vs. changes here:
* Of the 17 that have x-variables, 3 qnodes were correct in both versions, but one of those is different in either version (i.e. each one failed to select a qnode that the other version did). The new version guessed five more Qnodes, but none of those were accurate (hoping Mitchell's updates will help with that).
* Of the 3 that had claimers, 1 qnode was correct in both versions
* Of the 20 claims, 5 event qnodes were correctly selected in both versions. See note above.
* Of the 5 correct events, the original got 1 argument qnode + role correct while this version got 4. Ignoring role label accuracy, original: 4, new: 8.

**Pull request merged**